lib/js/components/SurveyQuestions/SurveyQuestionOpenEnded/SurveyQuestionOpenEnded.stories.ts

```diff
@@ -11,18 +11,8 @@ export default {
 const StoryTemplate: StoryFn<typeof SurveyQuestionOpenEnded> = (argTypes) => ({
 	components: { SurveyQuestionOpenEnded },
 	props: Object.keys(argTypes),
-	data() {
-		/* @todo revert valueData */
-		return {};
-	},
-	watch: {
-		value() {
-			// value is a prop here, Vue don't like modifying props, so we wrap it into valueData
-			this.valueData = this.value;
```

Comment:
because of this change, we're getting an error in console, when editing `value` in textarea

Reply:
you should use `updateArgs()`

Reply:
Can You check what I'm doing wrong with this `udpdateArgs`?

Reply:
See [`0318912` (#234)](https://github.com/bethinkpl/design-system/pull/234/commits/0318912445f27a15e9326185a0ba01363d54ce34)